src/liballoc/lib.rs

```diff
@@ -70,7 +70,6 @@
        test(no_crate_inject, attr(allow(unused_variables), deny(warnings))))]
 #![no_std]
 #![needs_allocator]
-#![cfg_attr(not(stage0), deny(warnings))]
```

Comment:
Ah can these actually stay? They're intended to emulate what the makefiles did with `-D warnings`.

Although I also personally `git grep stage0` to find what to remove on new snapshots. I would personally be in favor of changing this to `#![deny(warnings)]` (e.g. in all stages), but others may feel differently